pkg/target/target_template_source.go

```diff
@@ -288,6 +307,11 @@ matches_namespaces(match) {
   not has_field(match, "namespaces")
 }
 
+# Always match cluster scoped resources
+matches_namespaces(match) {
```

Comment:
We run the risk of a 1:1 error here. Also, per elsewhere, we may want to have split behavior for cluster-scoped resources between the `namespaces` and `excludedNamespaces` selectors.